torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1200,6 +1200,39 @@ def sample_inputs_svd(op_info, device, dtype, requires_grad=False):
 def sample_inputs_linalg_svd(op_info, device, dtype, requires_grad=False):
     return _sample_inputs_svd(op_info, device, dtype, requires_grad, is_linalg_svd=True)
 
+def sample_inputs_eig(op_info, device, dtype, requires_grad=False):
```

Comment:
Could you also add 0x0, 0xSxS and 2xSxS input matrices?

Reply:
No, I cannot. `torch.eig` does not support batching.